recipes/libcufile/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set name = "libcufile" %}
+{% set version = "1.5.0.59" %}
+ {% set platform = "linux-x86_64" %}  # [linux64]
+ {% set platform = "linux-ppc64le" %}  # [ppc64le]
+ {% set platform = "linux-sbsa" %}  # [aarch64]
+ {% set platform = "windows-x86_64" %}  # [win]
+ {% set extension = "tar.xz" %}  # [not win]
+ {% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 222c49b29afb544c69542c4e658b0a1bd01fd2306f2136ad6990ccfff482a097  # [linux64]
+  patches:
+    - Makefile.patch
+
+build:
+  # uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # it is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # see https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  # if the installation is complex, or different between unix and windows, use separate bld.bat and build.sh files instead of this key.
+  # by default, the package will be built for the python versions supported by conda-forge and for all major oss.
+  # add the line "skip: true  # [py<35]" (for example) to limit to python 3.5 and newer, or "skip: true  # [not win]" to limit to windows.
+  # more info about selectors can be found in the conda-build docs: 
+  # https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors
+  #script: {{ python }} -m pip install . -vv
+  number: 0
+  skip: true  # [osx]
+  skip: true  # [ppc64le]
+  skip: true  # [aarch64]
+  skip: true  # [win]
+
+outputs:
+  - name: libcufile
+    files:
+      - lib/libcufile*.so.*
+      - etc/cufile.json
```

In terms of tests, we can add some file tests like this here and to steps below

```suggestion
      - etc/cufile.json
    test:
      commands:
        - test -f $PREFIX/lib/libcufile*.so.*
        - test -f $PREFIX/etc/cufile.json
```

---

Thanks! Added

---

Looks like test like this for `gds-tools` fail

```
Tests failed for gds-tools-1.5.0.59-ha770c72_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
WARNING:conda_build.build:Tests failed for gds-tools-1.5.0.59-ha770c72_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
TESTS FAILED: gds-tools-1.5.0.59-ha770c72_0.conda
```

Not sure what `gds-tools` is, but I'm guessing `test -f` is failing because we didn't necessarily build this library?